crates/sui-adapter/src/in_memory_storage.rs

```diff
@@ -26,7 +26,9 @@ impl BackingPackageStore for InMemoryStorage {
 
 impl ParentSync for InMemoryStorage {
     fn get_latest_parent_entry_ref(&self, object_id: ObjectID) -> SuiResult<Option<ObjectRef>> {
-        debug_assert!(!self.persistent.contains_key(&object_id));
+        if let Some(obj) = self.persistent.get(&object_id) {
```

Comment:
Why was this changed? Was this a bug previously?

Reply:
Err wrong PR (though you will see that bug soon :) ). So it was probably just an untested codepath